engines/wintermute/base/base_file_manager.cpp

```diff
@@ -318,7 +318,7 @@ Common::SeekableReadStream *BaseFileManager::openPkgFile(const Common::String &f
 
 bool BaseFileManager::hasFile(const Common::String &filename) {
 	if (scumm_strnicmp(filename.c_str(), "savegame:", 9) == 0) {
-		BasePersistenceManager pm(BaseEngine::instance().getGameTargetName());
+		BasePersistenceManager pm(BaseEngine::instance().getGameTargetName().c_str());
```

Comment:
I guess why not.
Since Common::String inits to "" I believe the condition != "" to be equivalent to the nullptr-check.

Where I was at it I update initSave as well, to match initLoad.